codespell_lib/tests/test_basic.py

```diff
@@ -100,15 +100,15 @@ def test_basic(tmpdir, capsys):
     os.remove(fname)
 
     with open(op.join(d, 'bad.txt'), 'w') as f:
-        f.write('abandonned\nAbandonned\nABANDONNED\nAbAnDoNnEd')
-    assert cs.main(d) == 4
+        f.write('abandonned\nAbandonned\nABANDONNED\nAbAnDoNnEd\nabandonned\rAbandonned\r\nABANDONNED \nAbAnDoNnEd')  # noqa: E501
+    assert cs.main(d) == 8
     code, _, stderr = cs.main('-w', d, std=True)
     assert code == 0
     assert 'FIXED:' in stderr
     with open(op.join(d, 'bad.txt')) as f:
         new_content = f.read()
     assert cs.main(d) == 0
-    assert new_content == 'abandoned\nAbandoned\nABANDONED\nabandoned'
+    assert new_content == 'abandoned\nAbandoned\nABANDONED\nabandoned\nabandoned\rAbandoned\r\nABANDONED \nabandonned'  # noqa: E501
```

Comment:
```suggestion
    assert new_content == 'abandoned\nAbandoned\nABANDONED\nabandoned\nabandoned\rAbandoned\r\n ABANDONED \nabandoned'  # noqa: E501
```

Reply:
Oops, good catch on the typo. I've kept that, but updated the rest to match my tweak above.
```suggestion
    assert new_content == 'abandoned\nAbandoned\nABANDONED\nabandoned\nabandoned\rAbandoned\r\nABANDONED \n abandoned'  # noqa: E501
```